app/assets/stylesheets/pages/_index.scss

```diff
@@ -1 +1,5 @@
-@import "home";
\ No newline at end of file
+@import "home";
+
+.wrapper {
+  margin-top: 80px;
```

Comment:
Use variable of navbar : $navbar-height and $navbar-padding-vertical (add twice)